doc/feature/pointDataEvolution.md

```diff
@@ -0,0 +1,85 @@
+* As output, users will obtain a series of values corresponding to the evolution
+  of the data at that specific point over time (e.g. one value per simulation
+  frame).
+* The computation should be independent from the voxelization step, as the
+  point for which the values will be computed should not be limited to discrete
+  voxels, but a continuous 3D space.
+* Optionally, it could use ZeroEQ to stream the data to an external component,
+  like a GUI showing a 2D graph that is updated dynamically as the values are
+  being computed.
+
+## Implementation
+
+The core of the computation is already present in Fivox in the form of functors,
+that take a 3D point as a parameter and, based on the use case and the different
+types of functors available, return a value that is the result of evaluating the
+existing events.
+
+To make the computation of these values easy from a user point of view, we need
+to expose the usage of the functors in a separate step as an independent
+application. We could either overload the 'voxelize' app, or have a new and more
+simple one. Since we don't want to depend on the voxelization step or the
+volume creation for the computation of such values, we will create a new
+application that will have some parts in common with other apps, such as the
+volume URI evaluation or the frame/time ranges to compute. For that reason, the
+'voxelize' app might need to be refactored to extract the common parts into a
+superclass that will be used as base for all the applications.
+
+To output the resulting values, there are several possibilities:
+
+1. Writing to file
+
+2. Plot directly as a 2D graph
+
+3. Stream through ZeroEQ
+
+Option 2 would require the integration of a plotting library in C++. Option 3
+adds complexity that is not needed now, as it would only make sense for
+the streaming case, which is not a priority at the moment.
+
+For the sake of simplicity, we will start with a basic use case in which we
+write from the C++ app to a file and then add a separate step to create the 2D
+plot using a simple library in Python (e.g. matplotlib).
+
+The usage of the C++ app would be as:
+
+    dataEvolution --volume "fivox://BlueConfig?target=Column&dt=0.1" --point [x, y, z] --frame[s] "0 10" [or alternatively, --time[s] "0 100"] --output data.txt
+
+And the Python plotting tool:
+
+    plot2D --input data.txt --output graph.png
+
+Potentially, both steps could be combined into one wrapping tool that runs both
+steps one after the other.
+
+## File format
+
+The resulting plot should reflect the time evolution on the X axis, and the data
+values on Y. Therefore, the C++ tool will need to generate files in the
+following format:
+
+* One line per timestep
+* Each of the lines containing the timestamp (ms) and value (output unit),
+  separated by whitespaces
+* Potentially, a header containing the information about the way the data was
+  generated (e.g. the Fivox volume URI, the number of frames, etc)
+
+## Issues
+
```

Comment:
For me the first question to answer is, why a new application is created? You already gave an answer above (simplicity) but I think that explaining why that decision is taken is more important that the name of the application.